src/encoded/helpers/__init__.py

```diff
@@ -0,0 +1 @@
+# package
```

Comment:
Remove all these # package things.  I know it's in other ones but they don't have a purpose.  

You can add a doc string at the top explaining the reason for the package.
```
"""
Container for code shared by multiple scripts or modules.
"""
```